packages/devtools_app/lib/src/timeline/flutter/timeline_controller.dart

```diff
@@ -0,0 +1,350 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
can you comment on which parts of this file are new or upload a baseline version that is the code it was forked from?

Reply:
If you look at the latest commit, you can see the changes against the forked version.